cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateNamespaceCommand.java

```diff
@@ -35,6 +37,12 @@
   private static final String SUCCESS_MSG = "Namespace '%s' created successfully.";
 
   private final NamespaceClient namespaceClient;
+  private static final String NAMESPACE_PRINCIPAL = "principal";
+  private static final String NAMESPACE_KEYTAB_PATH = "keytab-path";
```

Comment:
shouldn't these keys be consistent with the keys used, when submitting the json via curl?